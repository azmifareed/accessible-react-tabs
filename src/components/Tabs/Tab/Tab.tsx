import { ReactNode } from 'react';

export interface ITabProps {
  children?: ReactNode;
  label: string;
  id: string;
  isSelected?: boolean;
}

export function Tab (props: ITabProps) {
  return null;
}
