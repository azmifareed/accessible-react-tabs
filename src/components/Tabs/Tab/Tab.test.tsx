import React from 'react';
import { render } from '../../../utils/test-utils';
import { Tab } from './Tab';


test('renders Tab correctly', () => {
  const component = render(<Tab id="test" label="test label">test child</Tab>)
  expect(component.baseElement).toMatchSnapshot();
});
