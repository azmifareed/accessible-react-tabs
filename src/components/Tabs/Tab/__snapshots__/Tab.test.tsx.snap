// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Tab correctly 1`] = `
<body>
  <div />
</body>
`;
